src/test_suite/test_runner_main.ts

```diff
@@ -46,6 +46,10 @@ export async function testRunnerMain(
     if (args.d) {
         dataPath = args.d;
     }
+    if (world) {
```

Comment:
This is fine for this commit, but let's see if we can refactor code that uses prix-fixe so that the test_runner_main can create the World. Having one way to create the world will reduce complexity.

The system will be better when (dataPath !== undefined) has only one meaning.